ui/lib/core/addon/utils/client-count-utils.js

```diff
@@ -87,3 +71,89 @@ export const homogenizeClientNaming = (object) => {
   }
   return object;
 };
+
+const flattenDataset = (object) => {
+  if (Object.keys(object).includes('counts') && object.counts) {
```

Comment:
This seems verbose, since if `counts` does not exist on object the following will by falsy, as well as if object is undefined or null. Are we trying to safeguard against other types? 
```suggestion
  if (object?.counts) {
```

Reply:
I've gotten inconsistent data back...in some cases the `counts` and `new_clients` keys exist on the object but are `null`, in other cases the keys don't exist at all. Not sure if this answers your question?

Reply:
In that case I _think_ `if (object?.counts)` will suffice, but a great opportunity to spot check with tests!